addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/CrossSiteScriptingScanRuleUnitTest.java

```diff
@@ -448,6 +489,44 @@ protected Response serve(IHTTPSession session) {
         assertThat(alertsRaised.get(0).getConfidence(), equalTo(Alert.CONFIDENCE_MEDIUM));
     }
 
+    @Test
+    void shouldReportXssInBodyWithFilteredScripts() throws NullPointerException, IOException {
+        String test = "/shouldReportXssInBody/";
```

Comment:
Here and in `shouldReportXssInSpanContentWhenBypassingTagCloseAttacksAtLowThreshold` the path does not match the test name, not that it needs to IMHO we could just use a generic name always, the tests run in isolation the paths don't have to be unique nor they give any info since the failures usually don't include the URL path of the test.

Reply:
Done